assets/stylesheets/components/_media-objects.scss

```diff
@@ -0,0 +1,172 @@
+/**
+** Media Objects
+** 1. Base layout
+** 2. Text
+** 3. Images
+** 4. Overrides
+** 5. Connect-specific layout
+**/
+
+
+
+
+// Base layout
+.media {
+  border-top: 1px solid rgba(darken($cr-gray-lighter, 10), .4);
+  cursor: pointer;
+  display: flex;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+
+  > div:last-child {
+    margin-left: 1rem;
+  }
+
+  .media-body {
+    flex-grow: 1;
+    width: initial;
+  }
+}
+
+
+
+
+// Text
+.media-heading {
+  @extend .component-header;
+  color: $cr-gray-dark;
+  margin-bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  &.media-heading-link {
+    color: $cr-cyan;
+  }
+}
+
+.say-hi {
+  .media-heading &,
+  .media-heading-stacked & {
+    color: $cr-gray-dark;
+    font-family: $base-font-face;
+    font-size: $font-size-base;
+    text-transform: initial;
+    padding-bottom: .5rem;
+  }
+}
+
+.media-heading-stacked {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+
+  h1 {
+    @extend .component-header;
+    color: $cr-white;
+    font-size: $font-size-h1;
+    margin: 0;
+  }
+}
+
+.media-meta {
+  @extend .control-label;
+  color: $cr-gray-light;
+
+  .joined_by {
+    margin-left: .5rem;
+  }
+}
+
+
+
+
+// Images
+// Create custom .img-size-* classes at 16px base
+@for $i from 1 through 6 {
```

Comment:
This is already supported [here](https://github.com/crdschurch/crds-styles/blob/development/assets/stylesheets/components/_images.scss#L12-L19).